static/sass/_pattern_navigation-dropdown.scss

```diff
@@ -0,0 +1,319 @@
+    @media (max-width: $breakpoint-navigation-threshold) {
+      display: none !important;
+    }
+  }
+
+  // overlay
+  .dropdown-window-overlay {
+    @include vf-animation(opacity, brisk, out);
+
+    background-color: rgba(17, 17, 17, 0.4);
+    height: 100%;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 7;
+  }
+
+  .dropdown-window {
+    @include vf-animation(all, brisk, out);
+
+    background-color: #262626;
+    box-shadow: 0 1px 32px 1px transparentize($color-dark, 0.8);
+    flex-direction: column;
+    position: absolute;
+    width: 100%;
+    z-index: 11;
+
+    &__content-container {
+      @extend %fixed-width-container;
+
+      @media (max-width: $breakpoint-navigation-threshold) {
+        padding: 0;
+      }
+    }
+  }
+
+  .dropdown-window__content {
+    color: $color-light;
+    padding: 0;
+
+    [hidden="true"] {
+      display: none;
+    }
+
+    .p-side-navigation {
+      @media (min-width: $breakpoint-navigation-threshold) {
+        height: 100%;
+        padding: 2rem 0;
+        position: relative;
+
+        &::after {
+          background: #262626;
+          bottom: 0;
+          content: "";
+          display: block;
+          left: -50vw;
+          position: absolute;
+          right: 0;
+          top: 0;
+        }
+      }
+
+      &__drawer {
+        height: 100%;
+        position: relative;
+
+        @media (max-width: $breakpoint-navigation-threshold) {
+          max-width: none;
+          position: initial;
+          transform: none;
+        }
+      }
+
+      &__link {
+        border-bottom: $dropdown-border;
+        padding: 0.75rem 2rem 0.75rem 1.5rem;
+
+        @media (min-width: $breakpoint-navigation-threshold) {
+          padding: 0.25rem 1rem 0.25rem 0.5rem;
+        }
+
+        &::before,
+        &:focus::before {
+          display: none;
+        }
+
+        &::after {
+          @extend %icon;
+          @include vf-icon-chevron;
+
+          bottom: 0;
+          content: "";
+          margin: auto;
+          position: absolute;
+          right: 1rem;
+          top: 0;
+          transform: rotate(-90deg);
+
+          @media (min-width: $breakpoint-small) {
+            right: 1.5rem;
+          }
+
+          @media (min-width: $breakpoint-navigation-threshold) {
+            display: none;
+          }
+        }
+
+        &.is-active {
+          background-color: #2d2d2d !important;
+        }
+      }
+
+      &__item:last-child {
+        .p-side-navigation__link {
+          border: 0;
+        }
+      }
+    }
+  }
+
+  .dropdown-window__footer {
+    padding: 0 1rem;
+
+    @media (min-width: $breakpoint-navigation-threshold) {
+      align-items: start;
+      border-top: $dropdown-border;
+      display: flex;
+      justify-content: space-between;
+      padding: 1rem 0;
+    }
+
+    [class*="p-button"] {
+      background: transparent;
+      margin: 0;
+    }
+  }
+
+  .dropdown-window__side-panel {
+    margin-top: 1rem;
```

Comment:
I believe you can replace all the `1rem` with `$spv-large` https://vanillaframework.io/docs/migration-guide-to-v3